File: cpp/lista/retangulo.cpp
/*Problema "retangulo"
Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor
da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos.*/


#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main() {
    
    cout << fixed << setprecision(4);

    double base, height, area, perimeter, diagonal;

    cout << "Base do retangulo: ";
    cin >> base;
    
    cout << "Altura do retangulo: ";
    cin >> height;

    area = base * height;
    perimeter = (base * 2.0) + (height * 2.0);
    
    diagonal = sqrt(pow(base, 2.0) + pow(height, 2.0));

    cout << "AREA DO RETANGULO = " << area << endl;
    cout << "PERIMETRO DO RETANGULO = " << perimeter << endl;
    cout << "DIAGONAL DO RETANGULO = " << diagonal << endl;

    return 0;
}

File: java/01-estrutura-sequencial/Retangulo.java

import java.util.Locale;
import java.util.Scanner;

public class Retangulo {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double base, height, area, perimeter, diagonal;
		
		System.out.print("Base do retangulo: ");
		base = sc.nextDouble();
		System.out.print("Altura do retangulo: ");
		height = sc.nextDouble();
		
		area = base * height;
		perimeter = (base * 2.0) + (height * 2.0);
		diagonal = Math.sqrt(Math.pow(base, 2.0) + Math.pow(height, 2.0));
		
		System.out.println("AREA: " + String.format("%.4f", area));
		System.out.println("PERIMETRO: " + String.format("%.4f", perimeter));
		System.out.println("DIAGONAL: " + String.format("%.4f", diagonal));
				
		sc.close();		

	}

}
